File: AdminPanal/AdminPanal.Web/Modules/ServerTypes/AylaStoreModule/ProductImageForm.ts
import { IntegerEditor, StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface ProductImageForm {
    ProductId: IntegerEditor;
    Image: StringEditor;
}

export class ProductImageForm extends PrefixedContext {
    static formKey = 'AylaStoreModule.ProductImage';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ProductImageForm.init)  {
            ProductImageForm.init = true;

            var w0 = IntegerEditor;
            var w1 = StringEditor;

            initFormType(ProductImageForm, [
                'ProductId', w0,
                'Image', w1
            ]);
        }
    }
}

File: AdminPanal/AdminPanal.Web/Modules/AylaStoreModule/ProductImage/ProductImageForm.cs
using Serenity.ComponentModel;

namespace AdminPanal.AylaStoreModule.Forms;

[FormScript("AylaStoreModule.ProductImage")]
[BasedOnRow(typeof(ProductImageRow), CheckNames = true)]
public class ProductImageForm
{
    public int ProductId { get; set; }
    public string Image { get; set; }
}